packages/app/src/overview/components/update-os-modals/update-available-modal/caution-section.component.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { defineMessages } from "react-intl"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+
+const messages = defineMessages({
+  updateAvailableCautionSectionTitle: {
+    id: "module.overview.updateAvailableCautionSectionTitle",
+  },
+  updateAvailableSequenceUpdateDescription: {
+    id: "module.overview.updateAvailableSequenceUpdateDescription",
+  },
+  updateAvailableSingleUpdateDescription: {
+    id: "module.overview.updateAvailableSingleUpdateDescription",
+  },
+})
+
+interface CautionSectionProps {
+  isSingleRelease: boolean
+}
+
+const CautionSectionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 2.4rem;
+
+  p {
+    text-align: left;
+  }
+`
+
+export const CautionSection: FunctionComponent<CautionSectionProps> = ({
+  isSingleRelease,
+}) => {
+  return (
+    <CautionSectionContainer>
+      <Text
+        displayStyle={TextDisplayStyle.Headline5}
+        color="primary"
+        message={messages.updateAvailableCautionSectionTitle}
+      />
+      {isSingleRelease ? (
+        <Text
+          displayStyle={TextDisplayStyle.Label}
+          color="primary"
+          message={
+            messages.updateAvailableSingleUpdateDescription
+          }
+        />
+      ) : (
+        <Text
+          displayStyle={TextDisplayStyle.Label}
+          color="primary"
+          message={
+            messages.updateAvailableSequenceUpdateDescription
+          }
+        />
+      )}
```

Comment:
Small thing, but shorter + less code repetition
```suggestion
      <Text
        displayStyle={TextDisplayStyle.Label}
        color="primary"
        message={
          isSingleRelease
            ? messages.updateAvailableSingleUpdateDescription
            : messages.updateAvailableSequenceUpdateDescription
        }
```

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/195f2c0333baf150f87d5d5973144480eb02e768